test/specs/upload/conf/upload/schema.json

```diff
@@ -1,152 +1,299 @@
-          "foreign_key_columns": [
-            {
-              "table_name": "file",
-              "schema_name": "upload",
-              "column_name": "fk_id"
-            }
-          ],
-          "annotations": {},
-          "referenced_columns": [
-            {
-              "table_name": "referee",
-              "schema_name": "upload",
-              "column_name": "id"
+    "tables": {
+        "file": {
+            "comment": "asset/object",
+            "kind": "table",
+            "keys": [
+                {
+                    "comment": null,
+                    "annotations": {},
+                    "unique_columns": [
+                        "id"
+                    ]
+                }
+            ],
+            "entityCount": 0,
+            "foreign_keys": [{
+                "comment": null,
+                "foreign_key_columns": [
+                    {
+                        "table_name": "file",
+                        "schema_name": "upload",
+                        "column_name": "fk_id"
+                    }
+                ],
+                "annotations": {},
+                "referenced_columns": [
+                    {
+                        "table_name": "referee",
+                        "schema_name": "upload",
+                        "column_name": "id"
+                    }
+                ]
+            }],
+            "table_name": "file",
+            "schema_name": "upload",
+            "column_definitions": [
+                {
+                    "comment": null,
+                    "name": "id",
+                    "default": null,
+                    "nullok": false,
+                    "type": {
+                        "typename": "serial4"
+                    },
+                    "annotations": {}
+                },
+                {
+                    "comment": null,
+                    "name": "fk_id",
+                    "default": null,
+                    "nullok": true,
+                    "type": {
+                        "typename": "int8"
+                    },
+                    "annotations": {}
+                },
+                {
+                    "comment": null,
+                    "name": "filename",
+                    "default": null,
+                    "nullok": true,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {}
+                },
+                {
+                    "comment": "asset/reference",
+                    "name": "uri",
+                    "default": null,
+                    "nullok": true,
+                    "type": {
+                        "typename": "text"
+                    },
+                    "annotations": {
+                        "tag:isrd.isi.edu,2017:asset": {
+                            "url_pattern" : "/hatrac/ermrestjstest/{{{_fk_id}}}/{{{_uri.md5_hex}}}",
```

Comment:
Should we change this hatrac url_pattern in this PR too? or should it be part of hatrac test cases?